scripts/build/README.md

```diff
@@ -0,0 +1,55 @@
+# Prettier Build Script
+
+## Usage
+
+```sh
+yarn build
+```
+
+## Flags
+
+### `--no-cache` flag
+
+Run script with `--no-cache` flag will clean up the cache directory (`.cache` in project root) before bundling files.
+
+```sh
+yarn build --no-cache
+```
+
+### `--playground` flag
+
+Run script with `--playground` flag will only build files needed for the website.
+
+```sh
+yarn build --playground
+```
+
+### `--print-size` flag
+
+To print the bundled file sizes:
+
+```sh
+yarn build --print-size
+```
+
+### `--minify` and `--no-minify` flags
+
+Force minify files:
+
+```sh
+yarn build --minify
+```
```

Comment:
Our build script will minify by default, right? Do we need to make this explicit?

Reply:
I think I'll use it with `--file` flag together, if I want debug build script.

Reply:
Okay, I have no better idea for this. But I think if this option is for debugging we should explicit it.

Reply:
https://github.com/prettier/prettier/pull/11858/commits/f136b1e876e2fe0c8ff37df9d65f81b1fb8a8308

I'm going to need this to debug #11861